src/v2/features/vs/equipment-select.tsx

```
@@ -5,12 +5,11 @@ import React, { useMemo } from 'react';
 import { Equipment } from 'src/models/enums';
 import { EquipmentType } from 'src/models/interfaces';
 import { StaticDataService } from 'src/services';
+import { DamageCalculatorService } from './damage-calculator-service';
```

ESLint

[error] There should be at least one empty line between import groups

(import-x/order)